build.sh

```
@@ -2,122 +2,34 @@
-
-#### Install zsh packages ####
-INSTALL_PACKAGE_LIST="$INSTALL_PACKAGE_LIST zsh-autosuggestions zsh-syntax-highlighting"
-
-#### Package for flatpak Steam ####
-INSTALL_PACKAGE_LIST="$INSTALL_PACKAGE_LIST steam-devices" # Coders also game
-
-
-#### Actually install packages ####
 sed -i "s/^enabled=.*/enabled=1/" /etc/yum.repos.d/rpmfusion-nonfree-steam.repo
-rpm-ostree install $INSTALL_PACKAGE_LIST
+rpm-ostree install $(cat $BUILD_FILES_DIR/install-pkgs)
```

Shellcheck

[warning] Quote this to prevent word splitting.

(SC2046)